white/continuous-security-improvement.md

```diff
@@ -0,0 +1,10 @@
+# Continuously Improve your Security Belt
+
+Security is like a big elephant. You cannot eat it as a whole, but you can slice it and continuously eat small pieces. To make this happen, ensure that the Product Owner continuously prioritizes security belt activities within the product backlog. Security belt activities make good pieces because they are of reasonable size and have a defined output. Celebrate all your implemented security belt activities.
```

Comment:
```suggestion
Security is like a big elephant. You cannot eat it as a whole, but you can slice it and continuously eat small pieces. To make this happen, ensure that the Product Owner continuously prioritizes security belt activities for the next belt within the product backlog. Security belt activities make good pieces because they are of reasonable size and have a defined output. Celebrate all your implemented security belt activities.
```